aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -289,6 +309,13 @@ module aptos_framework::fungible_asset {
         exists<FungibleStore>(store)
     }
 
+    #[view]
+    /// Return whether the provided address has a concurrent fungible balance initialized,
+    /// at the fungible store address.
+    public fun concurrent_fungible_balance_exists(store: address): bool {
+        exists<ConcurrentFungibleBalance>(store)
+    }
```

Comment:
```suggestion
    #[view]
    /// Return whether the provided address has a concurrent fungible balance initialized,
    /// at the fungible store address.
    public fun concurrent_fungible_balance_exists(store: address): bool {
        exists<ConcurrentFungibleBalance>(store)
    }


    inline fun concurrent_fungible_balance_exists_inline(store: address): bool {
        exists<ConcurrentFungibleBalance>(store)
    }

    inline fun store_exists_inline(store: address): bool {
        exists<FungibleStore>(store)
    }
```

How about adding inline versions here as well, which can be used in control flow statements?